.eslintrc.yml

```diff
@@ -27,6 +27,7 @@ rules:
     - off
   "@typescript-eslint/explicit-module-boundary-types":
     - error
+    - allowArgumentsExplicitlyTypedAsAny: true
```

Comment:
See comments below on this. I'd imagine we can unset this again. Might require some assertions but that's probably safer anyways.